docs/src/donating.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
+
```

Comment:
This page is linked via the top navbar so it does not need to be included in a TOC